requirements/base.txt

```diff
@@ -38,7 +38,7 @@ pyjwt[crypto]==2.5.0
     #   social-auth-core
 python3-openid==3.2.0
     # via social-auth-core
-pytz==2022.2.1
+pytz==2022.4
```

Comment:
No breaking change affecting the current implementation https://github.com/stub42/pytz/commit/8eeefc360913ce56f8759c33b31e00542a0d3078